### PR TITLE
[Playground] [Frontend] FCE updated to v0.3.0, minor improvements

### PR DESCRIPTION
**Pull request by nausharipov**
FCE updated to v0.3.0, minor improvements

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by nausharipov** [comment]
R: @alexeyinkin

File: playground/frontend/lib/modules/messages/parsers/messages_parser.dart [hunk 1]
```diff
@@ -44,7 +43,7 @@ class MessagesParser {
         if (map is Map) {
           return _tryParseMap(map);
         }
-      } on FormatException catch (ex) {
+      } on FormatException catch (_) {
```
Comment (alexeyinkin) [inline playground/frontend/lib/modules/messages/parsers/messages_parser.dart:46]: https://dart.dev/tools/linter-rules#no_wildcard_variable_uses

**Review by alexeyinkin — APPROVED** [review]
LGTM (internal review).

**Comment by pabloem** [comment]
frontend test broken?

**Comment by pabloem** [comment]
lgtm!

**Comment by alexeyinkin** [comment]
Wow, I have just learned a great lesson: https://medium.com/@alexey.inkin/how-an-automated-merge-in-git-can-kill-you-ecfe697cc9f6
@pabloem do you know how to avoid silence in such conflicts? Can any non-default merge strategy break on such mismerge and not silently succeed?

**Pull request merged**